module1.js

```diff
@@ -80,7 +100,9 @@ function question5(number1, number2) {
 */
 
 function question6(array) {
+
   // TODO:
+  // return `The array contains ${array.length} animals. These are: ${array[]},${element2},${element3},${element4}.`;
```

Comment:
This was a good start - think about the array methods you could use to get the second part of the output string.